docs/index.rst

```diff
@@ -20,6 +20,23 @@ Pangeo
 A community platform for Big Data geoscience
 ============================================
 
+What is Pangeo?
+^^^^^^^^^^^^^^^
+
+Pangeo is first and foremost a community promoting open, reproducible, and scalable science.
+This community provides documentation, develops and maintains software, and deploys
+computing infrastructure to make scientific research and programming easier.
+The Pangeo software ecosystem involves
+open source tools such as xarray, iris, dask, jupyter, and many other packages.
+There is no single software package called "pangeo"; rather, the Pangeo project serves
+as a coordination point between scientists, software, and computing infrastructure.
+On this website, scientists can find guides for accessing data and performing analysis
+using these tools. 
+Developers and IT professionals can find
+instructions for deploying cluster or cloud-based infrastructures.
+We welcome you to get more involved by...
```

Comment:
Here it could be a simple:
"We welcome you to get more involved by joining the conversation on our GitHub repositories."
And then have a link to a dedicated "Get Involved" page (there are tiny bits here and there on various pages) where we can give more details. I owe a "beginner guide for github" and that could be a good place for it (where it is explained how GitHub is used also as a mailing list essentially)